d3a_interface/sim_results/__init__.py

```diff
@@ -50,6 +50,17 @@ def is_buffer_node_type(area):
     return area['type'] == "InfiniteBusStrategy"
 
 
+def is_house_node_type(area):
+    return get_unified_area_type(area) == "Area"
+
+
+def has_no_grand_child(area):
```

Comment:
```suggestion
def has_no_grand_children(area):
```

Reply:
Sorry again english grammar(not my mother tongue) but whats wrong with singular noun here? 
I'm checking not even a single grand kid for this target area.

Reply:
If the noun following the no is countable, it is preferred to use it in plural 
https://englishlessonsbrighton.co.uk/followed-singular-plural-noun/

Reply:
Thanks for the link, will adapt it accordingly.